docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -0,0 +1,133 @@
+| `enableTelemetry` | Boolean | No | Telemetry will be enabled when this property is set to `true`, otherwise no telemetry data will be gathered during the execution of the wrapper. | `false` |
+| `telemetryTracesBackend` | String | No | Determines to which backend the gathered tracing data will be forwarded to.<br>`NONE` indicates that the application will collect tracing data but this data will not be forwarded anywhere.<br>`XRAY` indicates that the traces will be collected by the AWS XRay Daemon.<br>`OTLP` indicates that the traces will be collected by the AWS OTEL COllector. | `NONE` |
+| `telemetryMetricsBackend` | String | No | Determines to which backend the gathered metrics data will be forwarded to.<br>`NONE` indicates that the application will collect metrics data but this data will not be forwarded anywhere.<br>`OTLP` indicates that the metrics will be collected by the AWS OTEL COllector. | `NONE` |
+| `telemetrySubmitTopLevel` | String | No | Determines which nesting strategy to adopt for traces about top level traces and nested traces.<br><br>See the [Nested tracing strategies using Amazon XRay] section for more details about nesting strategies.<br><br>`ALWAYS_TOP_LEVEL` means that every JDBC call to the driver will generate a trace with no direct parent trace attached to it. If there is already an open trace being recorded by the application, no driver traces will be created.<br>`ALWAYS_NESTED` means that the wrapper will assume that there is an open trace in the application and the driver traces will all be attached to that open trace. If the user selects this value for the property and there is no open trace in the user application, no trace will be created.<br>`TOP_LEVEL` means that the application will look if there is an open trace in the users application prior to record telemetry data. If there is a current open trace, the traces created will be attached to that open trace. If not, all telemetry traces created will be top level.<br>`NO_TRACE` means that no traces will be created during the wrapper execution. | `TOP_LEVEL` |
+
+## Nested tracing strategies using Amazon XRay
+
+As you could see in the previous section, the AWS JDBC Driver allows an user to determine which strategy for nested traces to use when using Telemetry.
+
+Traces are hierarchical entities and it might be that the user application has already an open trace in a given sequence of code that connects to the AWS JDBC Driver. In this case, the Telemetry feature allows users to determine which strategy to use for the Telemetry traces generated when using the driver.
```

Comment:
```suggestion
Traces are hierarchical entities and it might be that the user application already has an open trace in a given sequence of code that connects to the AWS JDBC Driver. In this case, the Telemetry feature allows users to determine which strategy to use for the Telemetry traces generated when using the driver.
```